meilisearch-http/src/analytics/segment_analytics.rs

```diff
@@ -182,6 +184,119 @@ impl super::Analytics for SegmentAnalytics {
     }
 }
 
+#[derive(Debug, Clone, Serialize)]
```

Comment:
Could you add a comment on this struct, to explain that this is the one sent in the analytics, please?